ccd-definition/CaseTypeTab.json

```diff
@@ -288,5 +288,16 @@
     "TabDisplayOrder": 4,
     "CaseFieldID": "submittedForm",
     "TabFieldDisplayOrder": 13
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "CasePeopleTab",
+    "TabLabel": "People in the case",
+    "TabDisplayOrder": 2,
+    "CaseFieldID": "applicants",
+    "TabFieldDisplayOrder": 4
```

Comment:
let's move all of this up to the other CasePeopleTab bits

Reply:
👍